src/js/datebuilder.js

```diff
@@ -0,0 +1,25 @@
+const dateBuilder = date => {
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+
+  const days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const currentDay = days[date.getDay()];
+  const currentDate = date.getDay();
+  const month = months[date.getMonth()];
+  const year = date.getFullYear();
+  return `${currentDay} ${currentDate} ${month} ${year}`;
+};
```

Comment:
Can you add test for this too?

Also, not that there's a bug with this function see below screenshot.

![image](https://user-images.githubusercontent.com/36057474/91668245-3a136400-eb03-11ea-92f7-e3e5ff7639a0.png)

Reply:
I'll ensure to debug this